_RELEASE/Assets/favoriteLevels.json

```diff
@@ -2,10 +2,7 @@
 {
 	"ids" : 
 	[
-		"LoFiOHv1_Jocii_Chill_Open_Hexagon_to_Calm_the_Vibe_1_LoFi01",
-		"ohvrvanilla_vittorio_romeo_cube_1_apeirogon",
-		"ohvrvanilla_vittorio_romeo_cube_1_commando",
-		"ohvrvanilla_vittorio_romeo_experimental_1_curvetest",
-		"ohvrvanilla_vittorio_romeo_experimental_1_curvetest2"
+		"ohvrvanilla_vittorio_romeo_experimental_1_arcadia",

```

Comment:
Favorite levels should be saved on a per-profile basis, but I'm fine with that being done for a future update. Also this file should be cleared up before merging.